src/main/java/taxi/filter/AuthenticationFilter.java

```diff
@@ -0,0 +1,45 @@
+package taxi.filter;
+
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.Set;
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+
+public class AuthenticationFilter implements Filter {
+    private final Set<String> allowedUrls = new HashSet<>();
+
+    @Override
+    public void init(FilterConfig config) {
+        allowedUrls.add("/");
+        allowedUrls.add("/register");
+        allowedUrls.add("/drivers/add");
```

Comment:
why we have 2 different endpoints to add driver: `/register` and `/drivers/add`?